app/views/admin/loans/timeline/_edit_project_step.html.slim

```diff
@@ -0,0 +1,43 @@
+div.panel-body
+
+  = simple_form_for([:admin, step],
+          remote: true,
+          html: { class: 'project-step-form' },
+          defaults: {html5: true}) do |f|
+
+    .error_notification
+      p.alert.alert-danger
+        button.close type="button" data-hide="alert" x
+        i.fa.fa-exclamation-circle>
+        = I18n.t(:error_notification)
+
+    = f.error_notification
+    input type="hidden" id="deleted_locales" name="project_step[deleted_locales]" value="[]"
+
+    .step_form_left.col-sm-8
+      - step.used_locales.each do |l|
+        .step-form-left-group.language-block data-locale="#{l}"
+          .clearfix
+            = f.input "locale_#{l}".to_sym, label: false, collection: locale_options(step.permitted_locales), wrapper: :step_form_left_2, input_html: {class: 'locale'}, include_blank: false
+            = f.input "summary_#{l}".to_sym, wrapper: :step_form_left_12, input_html: {class: 'summary'}, placeholder: I18n.t(:summary, locale: l)
```

Comment:
These very long lines need to be split somehow please. Our style guide says max of 100+/10 chars.